Mailgun_Subscriptions/Subscription_Form.php

```diff
@@ -116,14 +116,26 @@ protected function do_form_contents( $instance ) {
 			$this->print_solitary_list($list);
 			echo '</p>';
 		}
+		if ( !empty($instance['name']) ) {
+			echo '<p class="full-name">';
+			printf( '<label for="mailgun-full-name-%d">%s</label> ', $instance_counter, __('Full Name', 'mailgun-subscriptions') );
+			$default_name = '';
+			if ( is_user_logged_in() ) {
+				$user = wp_get_current_user();
+				$default_name = trim( sprintf('%s %s', $user->user_firstname, $user->user_lastname) );
```

Comment:
Maybe it would be better not to set a default for the name? Even setting it as display_name presumes that it is their actual name.